packages/files-ui/src/Components/Modules/FileBrowsers/SharedFoldersOverview.tsx

```diff
@@ -108,73 +119,65 @@ const SharedFolderOverview = () => {
   }
 
   return (
-    <article
-      className={classes.root}
-    >
-      <header className={classes.header}>
-        <Typography
-          variant="h1"
-          component="h1"
-          data-cy="shared-overview-header"
-        >
-          <Trans>Shared folders</Trans>
-        </Typography>
-        <div className={classes.controls}>
-        </div>
-      </header>
-      <button
-        onClick={() => {
-          !!encryptedEncryptionKey && filesApiClient.createBucket({
-            name: `Cat Bucket ${Date.now()}`,
-            encryption_key: encryptedEncryptionKey,
-            type: "share"
-          }).then((res) => {
-            console.log(res)
-            refreshBuckets()
-          })
-            .catch(console.error)
-        }}>
-        Create a shared &quot;Cat Bucket&quot;
-      </button>
-      {isLoadingBuckets && (
-        <div
-          className={clsx(classes.loadingContainer)}
-        >
-          <Loading size={24}
-            type="light" />
-          <Typography variant="body2"
-            component="p">
-            <Trans>Loading your shared folders…</Trans>
+    <>
+      <article
+        className={classes.root}
+      >
+        <header className={classes.header}>
+          <Typography
+            variant="h1"
+            component="h1"
+            data-cy="shared-overview-header"
+          >
+            <Trans>Shared folders</Trans>
           </Typography>
-        </div>
-      )}
-      {!isLoadingBuckets && (
-        <Table
-          fullWidth={true}
-          striped={true}
-          hover={true}
-        >
-          <TableHead className={classes.tableHead}>
-            <TableRow type="grid"
-              className={classes.tableRow}>
-              {desktop &&
+          <div className={classes.controls}>
+            <Button variant='outline'
+              onClick={() => setCreateShareModalOpen(true)}>
+              <PlusIcon />
+              Create a Share
```

Comment:
I'll add a many comments about this, but I think we need to be consistent and use "Shared folder" as much as we can.
Translation is missing
```suggestion
              <Trans>Create a Shared Folder</Trans>
```